index.js

```diff
@@ -21,31 +21,40 @@ module.exports = {
     }
   },
 
-  included(app, parentAddon) {
+  included() {
     this._super.included.apply(this, arguments);
 
-    var target = parentAddon || app;
-
-    // allow addon to be nested - see: https://github.com/ember-cli/ember-cli/issues/3718
-    if (target.app) {
-      target = target.app;
+    let app;
+
+    // Climb up the hierarchy of addons up to the host
+    // If the addon has the `_findHost()` method (in ember-cli >= 2.7.0) then use it
+    if (typeof this._findHost === 'function') {
+      app = this._findHost();
+    } else {
+      // Otherwise, use the copied `_findHost()` implementation
+      // https://github.com/ember-cli/ember-cli/blob/v2.15.1/lib/models/addon.js#L614-L625
+      let current = this;
+      do {
+        app = current.app || app;
+      } while (current.parent.parent && (current = current.parent));
     }
 
+    this.app = app;
+
     // see: http://ember-cli.com/extending/#broccoli-build-options-for-in-repo-addons
-    target.options = target.options || {};
+    app.options = app.options || {};
 
     // Build path to Bulma's sass paths
-    var bulmaPath = path.join(target.project.root, 'node_modules', 'bulma');
+    let bulmaPath = path.dirname(require.resolve('bulma'));
```

Comment:
@alexdiliberto Wondering if you got a chance to install on a fresh app (with this update)? Any issues?

Reply:
@crodriguez1a Yea, doing it this way is more reliable because it resolves to the users's filesystem instead of always pointing to a `node_modules` dir